helm/dashboards/charts/private_dashboards_mz/dashboards/shared/private/mimir-continuous-tests.json

```diff
@@ -0,0 +1,702 @@
+{
+  "annotations": {
+    "list": [
+      {
+        "builtIn": 1,
+        "datasource": {
+          "type": "grafana",
+          "uid": "-- Grafana --"
+        },
+        "enable": true,
+        "hide": true,
+        "iconColor": "rgba(0, 211, 255, 1)",
+        "name": "Annotations & Alerts",
+        "type": "dashboard"
+      }
+    ]
+  },
+  "editable": true,
+  "fiscalYearStartMonth": 0,
+  "graphTooltip": 0,
+  "id": 143,
+  "links": [],
```

Comment:
We need the same as https://github.com/giantswarm/dashboards/blob/7af76dcf2252e1f15095bdd6395931d35b8ba6af/helm/dashboards/charts/private_dashboards_mz/dashboards/shared/private/mimir-compactor-resources.json#L17